package.json

```diff
@@ -484,6 +484,11 @@
           "when": "view == objectExplorer && viewItem =~ /\\btype=(AggregateFunction|PartitionFunction|ScalarValuedFunction|StoredProcedure|TableValuedFunction|View)\\b/",
           "group": "MS_SQL@4"
         },
+        {
+          "command": "mssql.editData",
+          "when": "view == objectExplorer && viewItem =~  /\\btype=(Table)\\b/",
```

Comment:
if you gate the feature behind the "enable preview features" flag as well, then it'll be easier to check in changes as they're ready.  One thing we want to avoid is to wait until the feature is almost done, then have a huge PR because that's more difficult to effectively review.